projects/packages/videopress/src/client/admin/components/clipboard-button-input/index.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@automattic/jetpack-components';
+import { useCopyToClipboard } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { useState } from 'react';
+/**
+ * Internal dependencies
+ */
+import styles from './style.module.scss';
+import { ClipboardButtonInput } from './types';
+import type React from 'react';
+
+const noop = () => {
+	/* noop */
+};
+
+/**
+ * Video Details Card component
+ *
+ * @param {ClipboardButtonInput} props - Component props.
+ * @returns {React.ReactNode} - VideoDetailsCard react component.
+ */
+const VideoDetailsCard: React.FC< ClipboardButtonInput > = ( {
+	text,
+	value,
+	onCopy = noop,
```

Comment:
You don't need to use a `noop` function, now with optional chaining, you can use it.

```
onCopy?.();
```

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#syntax

Reply:
Awesome. Updated 🙇‍♂️